target/cheri-common/cheri_defs.h

```diff
@@ -154,7 +154,13 @@ typedef enum CheriTbFlags {
      * PCC spans the full adddress space and has base zero. This means we do
      * not need to perform bounds checks or subtract/add PCC.base
      */
-    TB_FLAG_PCC_FULL_AS = (1 << 7)
+    TB_FLAG_CHERI_PCC_FULL_AS = (1 << 7),
+    TB_FLAG_CHERI_PCC_WRITABLE = (1 << 8),
```

Comment:
Hmmm, I don't think we need to optimise for this, there are no instructions that write via PCC directly even in Morello, right?

Reply:
@LawrenceEsswood ?

Reply:
And perhaps TB_FLAG_CHERI_PCC_VALID should become TB_FLAG_CHERI_PCC_EXECUTABLE to be more uniform with these? TB_FLAG_CHERI_PCC_VALID suggests it's just about the tag, but it's tag+unsealed+executable.

Reply:
Yeah, I just added that for symmetry.